server/utils/file.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// GetPluginRootPath ToDo
+func GetPluginRootPath() string {
```

Comment:
Since there is only one func in this file, perhaps it is better to put in i18n.go? 3/5

Reply:
This func is used for getting logo image, so moving this to `i18n.go` makes little confusing.
https://github.com/matterpoll/matterpoll/pull/133/files#diff-4444f0caa92e553b5c018c908e90e5e6R107

I think that it had better to leave this here for now.

Reply:
Oh!  I naively assumed it was created for this single purpose.